lib/cli/watch-run.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const utils = require('../utils');
+const Context = require('../context');
+const Mocha = require('../mocha');
+
+/**
+ * Exports the `watchRun` function that runs mocha in "watch" mode.
+ * @see module:lib/cli/run-helpers
+ * @module
+ * @private
+ */
+
+/**
+ * Run Mocha in "watch" mode
+ * @param {Mocha} mocha - Mocha instance
+ * @param {Object} opts - Options
```

Comment:
Why does this no longer provide a default value ({}) as it did before?

Reply:
As well as defaults for the object properties?

Reply:
see "Description of the change"

Reply:
Then all object properties _must_ exist, right? So why are they marked as optional?

Reply:
Good point. Addressed in https://github.com/mochajs/mocha/pull/3930/commits/6f69d01b7ca008f3992b7d1f5537240a4f68b305